reactive/jenkins.py

```diff
@@ -177,3 +198,29 @@ def migrate_charm_data():
 def stop():
     service_stop("jenkins")
     status_set("maintenance", "Jenkins stopped")
+
+
+@when("apt.installed.jenkins")
+@when("reset.jenkins.home")
+def reset_jenkins_home():
+    status_set("maintenance", "Configuring JENKINS_HOME")
+
+    home = unitdata.kv().get("storage.dir")
+    if home is None:
+        home = '/var/lib/jenkins'
+    log("Setting Jenkins home to {}".format(home))
+    unitdata.kv().set("jenkins.home", home)
+    # re-import paths to force update
+    importlib.reload(paths)
```

Comment:
I don't understand why this is needed. Where does this branch change global variables defined in the paths module?

Note also that the global variables defined in the paths module are meant to be constants, and should not be changed at run time. If you want something to be variable, than please make it a regular variable that is passed around.